engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -190,14 +190,51 @@ object JesBackend {
   def jesLogFilename(key: BackendCallKey) = s"${jesLogBasename(key)}.log"
   def jesReturnCodeFilename(key: BackendCallKey) = s"${jesLogBasename(key)}-rc.txt"
   def globDirectory(glob: String) = s"glob-${glob.md5Sum}/"
-  def globOutputPath(callPath: Path, glob: String) = callPath.resolve(s"glob-${glob.md5Sum}/")
+  def rootPath(options: WorkflowOptions): String = options.getOrElse(GcsRootOptionKey, ProductionJesConfiguration.jesConf.executionBucket).stripSuffix("/")
+
+  implicit class JesEnhancedJobDescriptor(val jobDescriptor: BackendCallJobDescriptor) extends AnyVal {
```

Comment:
Yes, this is the good kind of implicit that helpfully adds methods to an existing type.  :smile:  The `JobDescriptor` was made elsewhere and must not be specific to JES or any other backend.